recipes/swig/all/conanfile.py

```diff
@@ -98,6 +98,7 @@ def _configure_autotools(self):
         ]
         if self.settings.os == "Linux":
             args.append("LIBS=-ldl")
+            autotools.flags.append("-DHAVE_UNISTD_H=1")
```

Comment:
I'm going to upstream a change to the CMakeLists.txt so we can hopefully start using that instead of autotools: https://github.com/conan-io/conan-center-index/issues/15324#issuecomment-1468079165

Reply:
Adding a separate issue to address it at some point / track the progress: https://github.com/conan-io/conan-center-index/issues/16557